.github/workflows/ci.yaml

```diff
@@ -0,0 +1,108 @@
+name: CI
+
+on:
+  push:
+    branches:
+    - use-github-actions
+
+env:
+  registry: ghcr.io
+
+jobs:
+  image:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: angular
+    permissions:
+      contents: read
+      packages: write
+    steps:
+    - name: Check out the repository
+      id: checkout
+      uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # v4.1.7
+
+    - name: Set up Node.js
+      id: setup-node
+      uses: actions/setup-node@1e60f620b9541d16bece96c5465dc8ee9832be0b # v4.0.3
+      with:
+        node-version: current
+        cache: npm
+        cache-dependency-path: angular/package-lock.json
+
+    - name: Run clean install
+      id: npm-ci
+      run: npm ci
```

Comment:
I reference these steps later in the file to include their outcomes in the email summary. If they fail, it would be odd to get an email with a list of steps without a single failure (only "skips") other than the overall workflow status. Should I remove the IDs anyway?

Reply:
I see, ok then - keep it no issue